drivers/memc/Kconfig

```diff
@@ -13,7 +13,7 @@ if MEMC
 
 config MEMC_INIT_PRIORITY
 	int "Initialization priority"
-	default 0
+	default 80
```

Comment:
This may not be a good idea as it changed priorities for all the memc drivers and other people may still want to early init. 
default 80 if MSPI may be better.

Reply:
You are right.